Makefile

```
@@ -1,15 +1,12 @@
-.PHONY: test test-cov lint lint-py lint-sh
+.PHONY: test test-cov lint lint-py
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)